util/dvsim/Testplan.py

```diff
@@ -11,17 +11,51 @@
 from pathlib import Path
 
 import hjson
+import logging as log
 import mistletoe
 from tabulate import tabulate
 
 
+class JobTime:
+    # Possible units.
+    units = ["h", "m", "s", "ms", "ns", "us", "ps", "fs"]
+
+    def __init__(self):
+        self.time = 0.0
+        self.unit = "s"
+
+    def __str__(self):
+        if self.time == 0:
+            return ""
+
+        try:
+            index = self.units.index(self.unit)
+        except ValueError as e:
+            log.debug("Unsupport JobTime unit: " + self.unit + f"\n{e}")
+            return ""
+
+        # Normalize the time unit.
+        # The conversion from `fs` to `ms` is 1000, but from `s` to `h` is 60.
+        normalized_time = self.time
+        time_interval = 1000.0 if index > 2 else 60.0
+        while index > 0 and normalized_time >= time_interval:
+            normalized_time = normalized_time / time_interval
```

Comment:
Maybe you could add `JobTime.divider = [60,] * 3 + [1000,] * 5`, So this line will be 
```python3
        while index > 0 and normalized_time >= divider[index]:
            normalized_time = normalized_time / divider[index]
```

Reply:
that is a great idea! Thanks Sri.